### DISCUSSION TITLE
Ethernet activity LED rapidly flashing when BLE enabled

### DISCUSSION THREAD

**Discussion by PDE-Paul** [h2zero/NimBLE-Arduino#696] [Q&A]
Hi All,

I hope this is the correct place to ask this question.

I have a design that uses the LAN8710A connected to the RMII interface of my ESP32, everything functions fine, I have no issues with running BLE with the Ethernet API.

My question is why does the Ethernet LED for Activity indication, starts rapidly flashing when I enable BLE, if I disable BLE the LED functionality returns to flashing only when there is Ethernet communications.

Is there a hash define I need to setup to prevent this?

Once again this rapid flashing is not causing any code instability, but it would be great to "fix" this.

Kind Regards
Paul

**Comment by h2zero** [comment]
That's a strange one lol, no idea why this would happen but I'm going to guess it's something to do with the Ethernet driver lower down. Nothing in the BLE stack here would affect this that I could think of.
